apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.$projectParam.environments/route.tsx

```diff
@@ -146,17 +147,38 @@ export default function Page() {
                   <Header2>API Keys</Header2>
                   <HelpTrigger title="How do I use API Keys and Endpoints?" />
                 </div>
-                <div className="mb-8 mt-4 flex gap-4">
-                  {environments.map((environment) => (
-                    <ClipboardField
-                      key={environment.id}
-                      fullWidth={false}
-                      secure
-                      value={environment.apiKey}
-                      variant={"primary/medium"}
-                      icon={<EnvironmentLabel environment={environment} />}
-                    />
-                  ))}
+                <div className="mb-8">
+                  <Paragraph variant="small" spacing>
+                    Secret API keys should be used on your server – they give
```

Comment:
What's the thinking behind calling these "Secret API keys" and not either "Private API keys" or "Server API keys"?

Reply:
I'm happy to do `Private` or `Server`. I took it from Stripe but everyone does this differently.